homeassistant/components/unifiprotect/light.py

```diff
@@ -0,0 +1,91 @@
+"""This component provides Lights for UniFi Protect."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from pyunifiprotect.data import Light
+
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS,
+    SUPPORT_BRIGHTNESS,
+    LightEntity,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .data import ProtectData
+from .entity import ProtectDeviceEntity
+
+_LOGGER = logging.getLogger(__name__)
+
+ON_STATE = True
```

Comment:
Is this used?

Reply:
I think it was left over from the 0.10.x version. Does not seem to be.